Sources/TSCBasic/Process.swift

```diff
@@ -392,7 +395,7 @@ public final class Process: ObjectIdentifierProtocol {
 
         let argv = CStringArray(arguments)
         let env = CStringArray(environment.map({ "\($0.0)=\($0.1)" }))
-        let rv = posix_spawnp(&processID, argv.cArray[0], &fileActions, &attributes, argv.cArray, env.cArray)
+        let rv = posix_spawnp(&processID, (argv.cArray[0] as Optional)!, &fileActions, &attributes, argv.cArray, env.cArray)
```

Comment:
won't `argv.cArray[0]!` work?

Reply:
Yeah it would, just thought this was a "nicer" solution :)

Reply:
Heh. The `as` operator is for typecasting but since the element of `cArray` are already optional, I suggest not doing this since it might be confusing to the reader.